src/index.html

```diff
@@ -17,6 +17,12 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
The use of 'stopwatch--speed-up' suggests that you are using a modifier from the BEM methodology. Ensure that this modifier is being used correctly according to the BEM rules. Modifiers should only be used to represent different states or versions of a block or an element.